tests/integration_tests/scheduler/test_generic_driver.py

```diff
@@ -123,27 +123,26 @@ async def finished(iens, returncode):
 @pytest.mark.flaky(reruns=5)
```

Comment:
Don't you want to remove the `(reruns=5)` thingy then?

Reply:
No, there is still residual flakyness.